Adapted version of upsteam commit:

commit fcaf399ec79c550884c5052d075919b7368bd104
Author: Anton Danilov <littlesmilingcloud@gmail.com>
Date:   Thu Jan 13 14:47:43 2011 -0800

    show selected bridge

diff -up bridge-utils-1.2/brctl/brctl_cmd.c.show_selected_bridge bridge-utils-1.2/brctl/brctl_cmd.c
--- bridge-utils-1.2/brctl/brctl_cmd.c.show_selected_bridge	2012-09-07 15:02:02.000000000 +0200
+++ bridge-utils-1.2/brctl/brctl_cmd.c	2012-09-07 15:03:52.000000000 +0200
@@ -559,8 +559,15 @@ static int show_bridge(const char *name,
 
 static int br_cmd_show(int argc, char *const* argv)
 {
+	int i;
+
 	printf("bridge name\tbridge id\t\tSTP enabled\tinterfaces\n");
-	br_foreach_bridge(show_bridge, NULL);
+	if (argc == 1)
+		br_foreach_bridge(show_bridge, NULL);
+	else
+		for(i = 2; i <= argc; i++)
+			show_bridge(argv[i - 1], NULL);
+
 	return 0;
 }
 
@@ -663,7 +670,8 @@ static const struct command commands[] =
 	  "<bridge> <port> <prio>\tset port priority" },
 	{ 3, "setportmcrouter", br_cmd_setportmcrouter,
 	  "<bridge> <port> <int>\tset port multicast router" },
-	{ 0, "show", br_cmd_show, "\t\t\tshow a list of bridges" },
+	{ 0, "show", br_cmd_show,
+	  "[ <bridge> ]\t\tshow a list of bridges" },
 	{ 1, "showmacs", br_cmd_showmacs, 
 	  "<bridge>\t\tshow a list of mac addrs"},
 	{ 1, "showstp", br_cmd_showstp, 
